### PR TITLE
imported objects are not persisting 

### PR DESCRIPTION
**Pull request by nikhilmandlik**
### Resolves:
resolves: #4470

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### CODE REVIEW

File: platform/import-export/src/actions/ImportAsJSONAction.js [hunk 1]
```diff
@@ -115,8 +116,7 @@ define(['zepto', 'objectUtils'], function ($, objectUtils) {
                 delete newModel.persisted;
 
                 newObj = this.instantiate(newModel, keystring);
-                newObj.getCapability("location")
-                    .setPrimaryLocation(tree[keystring].location);
+                this.openmct.objects.save(newModel);
```
Comment (akhenry) [inline platform/import-export/src/actions/ImportAsJSONAction.js:119]: I think child `.location` needs to be set here? That's what the `locationCapability` used to do.

**Review by akhenry — CHANGES_REQUESTED** [review]
Needs to be rebased on 1.8.1, and children need `.location` to be set. Otherwise looks good.

File: platform/import-export/src/actions/ImportAsJSONAction.js (hunk 1 above)
Comment (nikhilmandlik) [reply platform/import-export/src/actions/ImportAsJSONAction.js:119]: I think its part of JSON

Comment (nikhilmandlik) [reply platform/import-export/src/actions/ImportAsJSONAction.js:119]: so before start anything, `generateNewIdentifiers` does `rewriteId` which basically is `tree = JSON.stringify(tree).replace(new RegExp(oldIdKeyString, 'g'), newIdKeyString);`

**Review by akhenry — APPROVED** [review]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? Y
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Pull request merged**